src/components/HomePage/SearchArea/useUrlSearchParams.js

```diff
@@ -5,6 +5,8 @@ import { useLocation, useHistory } from "react-router-dom";
 
 import { throttle } from "lodash";
 
+const HISTORY_REPLACE_THROTTLE_DELAY = 350; // milliseconds
```

Comment:
```suggestion
const HISTORY_REPLACE_THROTTLE_DELAY_MS = 350;
```
I think we have this format in other places